.github/workflows/test.yml

```
@@ -121,8 +121,8 @@ jobs:
         # TODO: We don't *need* a matrix with single values,
         # but is it worth keeping for supporting multiple versions in the future?
         matrix:
-          node: [14]
-          npm: [6]
+          node: [14, 16]
+          npm: [6, 7, 8]
```

Node 16 requires npm7. Let's skip the Node 16 and npm 6 build from the matrix:
```suggestion
          npm: [6, 7, 8]
          exclude:
          - node: 16
            npm: 6
```

---

Ignore me. We still need to test npm 6, 7, 8 with Node 14 and Node 16.

We'll drop npm 6 once Node 14 reaches EOL.